### ISSUE TITLE
Focus on cell when move up and down, or tab and enter

### ISSUE THREAD

**Issue by dfengbgl** [nadbm/react-datasheet#86]
I added a scrollbar for the rows, however when I use keyboard to move up and down, the scrollbar does not move accordingly. It seem like it is because a focus event to the cell is missing.

**Comment by nadbm** [comment]
Yes you are correct, currently the page does not focus to a cell. Im guessing this is a fairly simple change to make. PR is welcome

**Comment by greatwillow** [comment]
I've just made a PR that should add this functionality @nadbm .  See: #92 .
